physics/mp_thompson.F90

```diff
@@ -846,4 +878,18 @@ subroutine mp_thompson_finalize(errmsg, errflg)
 
       end subroutine mp_thompson_finalize
 
+      subroutine get_niwfa(aerfld, nifa, nwfa, ncol, nlev)
```

Comment:
**PLEASE provide a full description** of what is done inside this subroutine.  This is a BIG deal and a bunch of numbers are shown but no explanations for anything.  In particular, I wish to see what atmospheric constituents comprise the ```aerfld``` array index going from 1 to 15 and the hard-wired numbers are some sort of mass conversions probably.  But a solid description really must be included here.

Reply:
In GOCART and MERRA2, aerosol are given as mixing ratio (kg/kg).  To convert from kg/kg to #/kg, the "unit mass" (mass of one particle) within the mass bins is calculated. A lognormal size distribution within aerosol bins is used to find the size based upon the median mass. NIFA is mainly summarized over five dust bins and NWFA over the other 10 bins. The parameters besides each bins are carefully tuned for a good performance of  the scheme.

Reply:
@AnningCheng-NOAA   So then PLEASE write a proper description as a comment.  This is not a burden and this is a huge help to those of us who know nothing about MERRA aerosols.  The community needs a proper description.  I will adamantly refuse to accept the PR if one is not included.  It could still be merged by someone without a description, but that would be shame.

Reply:
@AnningCheng-NOAA @gthompsnWRF The description has been added into the code.